src/commands/Admin/eval.js

```diff
@@ -63,15 +58,16 @@ module.exports = class extends Command {
 		} catch (error) {
 			if (!syncTime) syncTime = stopwatch.friendlyDuration;
```

Comment:
add `if (!type) type = new Type(error);` here

Reply:
Okay. That'll fix the type codeblock being empty when an error is thrown, too